specification/hybridcompute/resource-manager/Microsoft.HybridCompute/stable/2022-11-10/HybridCompute.json

```diff
@@ -0,0 +1,1622 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "HybridComputeManagementClient",
+    "description": "The Hybrid Compute Management Client.",
+    "version": "2022-11-10"
```

Comment:
>version [](http://example.com/codeflow?start=5&length=7)

There are a bunch of breaking changes reported on this PR. Please get a breakingchanges approval and ARM can review the changes afterwards. Once you have the approval please remove the "ARMChangesRequested" label from the PR to make the PR visible to the ARM reviewers

Reply:
@rkmanda Can you review the PR again? We got the permission from the breaking change review board and added the extensions field back based on their request